src/main/java/ua/bookstore/online/dto/shopping/cart/QuantityDto.java

```diff
@@ -0,0 +1,9 @@
+package ua.bookstore.online.dto.shopping.cart;
+
+import jakarta.validation.constraints.NotEmpty;
+
+public record QuantityDto(
+        @NotEmpty
+        int quantity
```

Comment:
You can use here @Min if you don`t have check for negative numbers for this

Reply:
Good point.